deck.go

```diff
@@ -70,6 +72,42 @@ func LoadDeck(dev *streamdeck.Device, base string, deck string) (*Deck, error) {
 		d.Widgets = append(d.Widgets, w)
 	}
 
+	watcher, err := fsnotify.NewWatcher()
+	if err == nil {
+		err = watcher.Add(path)
+		if err == nil {
+
+			go func() {
```

Comment:
The problem with this is that we'll keep spawning new watchers every time we load a deck, even if it's one that has been loaded previously. We'll either need a central monitor that watches decks and remembers all the existing watchers or a way to terminate previous launched watchers.

Reply:
I've updated this so watcher is stored in the deck struct and can be closed before LoadDeck now. It worked from my test as when I'm on a different deck it no longer reverts to main deck when I edit that one.